sys-apps/smc-sum/smc-sum-2.2.0.20190220-r1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_DATE="$(ver_cut 4)"
+MY_PN="${PN/smc-/}"
+MY_PV="$(ver_cut 1-3)"
+
+DESCRIPTION="Updates the BIOS and IPMI firmware and system settings on Supermicro mainboards"
+HOMEPAGE="https://www.supermicro.com"
+SRC_URI="${MY_PN}_${MY_PV}_Linux_x86_64_${MY_DATE}.tar.gz"
+
+LICENSE="supermicro"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
```

Comment:
I noticed the source is x86_64, does it really work on x86? Legit question and not a complaint: has it been keyworded by an arch team, or did you at least test it?

Reply:
Seems, you nailed it. I just give a try and it's indeed amd64 only. Dropping arch.